docs/source/en/model_doc/qwen2.md

```diff
@@ -35,8 +35,8 @@ In the following, we demonstrate how to use `Qwen2-7B-Chat-beta` for the inferen
 >>> from transformers import AutoModelForCausalLM, AutoTokenizer
 >>> device = "cuda" # the device to load the model onto
 
->>> model = AutoModelForCausalLM.from_pretrained("Qwen/Qwen1.5-7B-Chat", device_map="auto")
->>> tokenizer = AutoTokenizer.from_pretrained("Qwen/Qwen1.5-7B-Chat")
+>>> model = AutoModelForCausalLM.from_pretrained("Qwen/Qwen2-7B-Instruct", device_map="auto")
```

Comment:
Thanks for updating - can you update the lines above introducing the code too?

Reply:
Of Course, It's my honor to do so. Thank you sis.